.github/workflows/build.yml

```diff
@@ -0,0 +1,40 @@
+name: Build & Deploy
+
+on:
+  workflow_dispatch: {}
```

Comment:
Should this pass an input for the repo?

Reply:
Would be a good idea!
Will do in separate PR